odata-json-format/7 Structural Property.md

```diff
@@ -195,6 +196,8 @@ they are annotated with the
 control information, in which case they MUST conform to the type
 described by the control information.
 
+Collections within an untyped collection are themselves untyped 
```

Comment:
```suggestion
Collections directly contained within an untyped collection are themselves untyped 
```

Reply:
Isn't the collection value of an object property in an untyped collection also untyped?

If so, the "directly" would mislead me.

Reply:
Can't I specify `@type` control information even in an untyped collection?

Reply:
🤷🏻‍♂️

Reply:
a structured item within the untyped collection could have a collection-valued property whose @type was specified. so "directly contained" is more precise.

Reply:
Renamed to "directly contained"